README.md

```diff
@@ -3,6 +3,30 @@ description: User Guide For Data Browser
 ---
 
 # Data Browser User Guide
+
+- [Introduction](#introduction)
```

Comment:
Not sure if this is/will be published anywhere (I recall seeing something about Docbook?), but it might be worth it looking into auto-generating a table of contents. At least Jekyll (for GitHub Pages) [supports that](https://github.com/solid/solid.github.io/commit/96610b6a54011d19988c3d3c0e215ef0710417f4).

Reply:
It would be great to have the Tables of Contents to be automatically generated from the headings. I'll create an issue to track.

Reply:
That actually happens.